commit f44b704b63c6f482f6983a8f7f75afac763b4276
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Thu Nov 22 17:28:02 2018 +0000

    media: bcm2835-unicam: Pass through the colorspace on try_fmt
    
    The current colorspace was always returned from try_fmt for no
    good reason.
    Return what the source subdevice returns instead.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/media/platform/bcm2835/bcm2835-unicam.c b/drivers/media/platform/bcm2835/bcm2835-unicam.c
index 29c749a5293b..545e729c7f0f 100644
--- a/drivers/media/platform/bcm2835/bcm2835-unicam.c
+++ b/drivers/media/platform/bcm2835/bcm2835-unicam.c
@@ -761,11 +761,7 @@ static int unicam_try_fmt_vid_cap(struct file *file, void *priv,
 		unicam_info(dev, "Sensor trying to send interlaced video - results may be unpredictable\n");
 
 	v4l2_fill_pix_format(&f->fmt.pix, &sd_fmt.format);
-	/*
-	 * Use current colorspace for now, it will get
-	 * updated properly during s_fmt
-	 */
-	f->fmt.pix.colorspace = dev->v_fmt.fmt.pix.colorspace;
+
 	return unicam_calc_format_size_bpl(dev, fmt, f);
 }
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1224_f44b704b63c6f482f6983a8f7f75afac763b4276.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1224_f44b704b63c6f482f6983a8f7f75afac763b4276.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
